internal/exec/stages/files/passwd.go

```diff
@@ -41,6 +41,9 @@ func (s *stage) createPasswd(config types.Config) error {
 			"/etc/.pwd.lock",
 			"/home",
 			"/root",
+			// for OSTree-based systems (newer restorecon doesn't follow symlinks)
+			"/var/home",
+			"/var/roothome",
```

Comment:
Maybe also add `/var/usrlocal` and `/var/opt`?  But this LGTM too.

See also https://github.com/rhinstaller/anaconda/commit/43a84550e8f24945ff4bf89a3f8ef94d56a73d98

Reply:
Those should be fine, no? Any files/dirs that users add in there through their Ignition configs will be marked for relabeling as Ignition creates them.

Reply:
I'm thinking though of things that drop files in `/usr/local` and `/opt`.

Reply:
I mentioned this in the commit message:

> More specifically, `restorecon` will follow symlinks that are
part of a given path, but not if the target path is a symlink itself.

So e.g. any files dropped under `/usr/local` and `/opt` will be relabeled properly. (I also just double checked this on FCOS to be triple sure).

Reply:
Ah sorry, missed that.  LGTM then!